nf_core/pipeline-template/.github/workflows/awsfulltest.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
           access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
           compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
+          pipeline: ${{ github.repository }}
```

Comment:
Hmmm...where did this come from again?

Reply:
And should we add it to `awstest.yml` too?

Reply:
I saw it [here](https://github.com/nf-core/rnaseq/blob/7106bd792b3fb04f9f09b4e737165fa4e736ea81/.github/workflows/awsfulltest.yml#L26-L27) and yes, as you nicely spotted it should also be added to `awstest.yml`

Reply:
I have lost track of where we are with these actions. @ewels I could swear we tested this stuff and had already updated the pipeline template? I'm confused.

Reply:
Ok. Given this code is in rnaseq on `master` and the Actions seems to be working I think we are good here.

Reply:
These lines were deliberately removed and should not have been added back here. Sorry I missed the ping fellas..

Reply:
Context - removed in https://github.com/nf-core/tools/pull/1395

This is because the new version of the Tower CLI tool needs the fully qualified GitHub URL now, instead of user/repo (which is what `github.repository` gives). The nf-core/tower-action does this properly by default now, but by specifying these in the template here we overwrite those defaults and break it.

Fixed in https://github.com/nf-core/tools/pull/1488